### PR TITLE
Add advisers index

### PR DESCRIPTION
**Pull request by neoeno**
[Screenshot removed because I forgot this was public! [Look here](https://moneyadviceservice.slack.com/archives/cultivate/p1434025270000006)]

The 'add adviser' stuff — which is the next PR — introduces a few changes to this. I'm breaking apart a largeish branch into a few smaller PRs so sorry for the slight mixing of concerns here!

### CODE REVIEW

**Comment by darokel** [comment]
:+1:

File: app/views/dashboard/advisers/_advisers_table.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      </tr>
+    </thead>
+    <tbody>
+      <% advisers.each do |adviser| %>
+        <tr class="t-advisers-table-row">
+          <td class="t-adviser-reference-number"><%= adviser.reference_number %></td>
+          <td class="t-adviser-name"><%= adviser.name %></td>
+          <td>
+            <ul class="l-comma-separated-list">
+              <% adviser.qualifications.each do |q| %>
```
Comment (tomas-stefano) [inline app/views/dashboard/advisers/_advisers_table.html.erb]: Can you rename to qualification?

Comment (neoeno) [inline app/views/dashboard/advisers/_advisers_table.html.erb]: Good idea. It is done.

**Comment by tomas-stefano** [comment]
:+1:

**Comment by neoeno** [comment]
Thanks!

**Pull request merged**